src/main/resources/sqlline/manual.txt

```diff
@@ -2118,6 +2127,14 @@ colorScheme
 
 If chester/dark/dracula/light/obsidian/solarized/vs2010, then this scheme will be used for command/sql syntax highlighting. If default then there is no syntax highlighting.
 
+confirm
+
+If true, then user will be prompted for confirmation if commands matching with confirmPattern are executed.
+
+confirmPattern
+
+If default with confirm = true, user will be prompted for DROP and DELETE commands, otherwise, for the commands as matched in the pattern.
```

Comment:
Same as above.

Reply:
This is actaully true in this scenario as default value is DROP/DELETE